tcms/testplans/templates/testplans/get.html

```diff
@@ -6,6 +6,7 @@
 {% load attachments_tags %}
 
 {% block title %}TP-{{ object.pk }}: {{ object.name }}{% endblock %}
+{% block head %}<link rel="stylesheet" type="text/css" href="{% static 'testplans/style/toolbar.css' %}" media="screen" />{% endblock %}
```

Comment:
you can place this on a new line to make it more readable.

Also drop the trailing slash in the tag. HTML5 doesn't use this syntax.